Stirrup.py

```diff
@@ -237,6 +237,15 @@ def makeStirrup(l_cover, r_cover, t_cover, b_cover, f_cover, bentAngle, bentFact
             int((size - diameter) / amount_spacing_value), f_cover)
     rebar.Direction = FaceNormal.negative()
     rebar.Rounding = rounding
+
+    # Rotate Stirrups with Structure
+    structureAngle = math.degrees(structure.Placement.Rotation.Angle)
+    if structureAngle in {0.0, 180.0}:
+        stirrupAngle = structureAngle
+    else:
+        stirrupAngle = structureAngle - 90
+    rebar.Placement.Rotation.__setattr__("Angle", math.radians(stirrupAngle))
```

Comment:
Here don't need to use `__setter__` method. Use `rebar.Placement.Rotation.Angle = math.radians(stirrupAngle)`.